krux_boto/util.py

```diff
@@ -39,3 +40,29 @@ def get_instance_region():
         get_logger('krux_boto').warn('get_instance_region failed to get the local instance region')
         raise Error('get_instance_region failed to get the local instance region')
     return zone.rstrip(string.lowercase)
+
+
+# Region codes
+class RegionCode(Enum):
+    ASH = ('ASH', 'us-east-1')
+    PDX = ('PDX', 'us-west-2')
+    DUB = ('DUB', 'eu-west-1')
+    SIN = ('SIN', 'ap-southeast-1')
```

Comment:
Except if I were to do that, the user will have to guess what is the key. (e.g. Is it 'ASH' or 'ash'? Is it 'us-east-1' or 'us_east_1'?) I wanted to avoid that. I am not a fan of this solution, though. Wish Python has a better enum.